client/directives/file-filter.js

```diff
@@ -0,0 +1,40 @@
+const isFileInput = el => el.tagName === 'INPUT' || el.type === 'file';
+const isProduction = process.env.NODE_ENV === 'production';
+
+export const install = Vue => {
+  const warn = (msg, vm) => Vue.util.warn(`[v-filefilter]: ${msg}`, vm);
+  Vue.directive('filefilter', {
+    inserted(el, { value }, { context: vm }) {
+      if (!isFileInput(el)) {
+        return !isProduction &&
+          warn('Using directive on incompatible element; expected `input[type="file"]`.', vm);
+      }
+      if (!value) return !isProduction && warn('Missing required argument.', vm);
+      if (value === 'auto') {
+        if (!vm.$validator) return;
+        const specifiers = readSpecifiers(vm.$validator, { name: el.name });
+        el.accept = specifiers.join(',');
+        return;
+      }
+      if (!Array.isArray(value)) {
+        return !isProduction &&
+          warn('Invalid argument type received; expected an Array or a String.', vm);
```

Comment:
You're actually expecting Array or "auto" which is a String and makes this type 'error' technically correct but IMO might be misleading in a way that consumer might assume he can send any String like `'jpg,png'` which you didn't anticipate here.

Reply:
Indeed, so here I'm refactoring it 👷

Reply:
This should be fixed now 'cause `v-filefilter` now allows all valid `accept` configuration inputs.